### DISCUSSION TITLE
Is static scanning still a thing?

### DISCUSSION THREAD

**Discussion by kissu** [unocss/unocss#81] [Q&A]
[Windi's static scanning](https://windicss.org/guide/extractions.html#extractions) was a thing before, will it still be the case with Uno?

Since Vue3 do have [dynamic CSS](https://v3.vuejs.org/api/sfc-style.html#state-driven-dynamic-css), the mix of the 2 on runtime would be cool!
Of course, comes the cost of doing this on runtime.

**Comment by antfu** [comment]
There is not technical blockers preventing you from doing that, but why you need it tho?

**Reply by kissu** [reply]
Mainly because even if this kind of solution is possible: https://github.com/tailwindlabs/tailwindcss/discussions/3461#discussioncomment-329183
I'd rather use `text-{color}-500` for ease of use, without having to rely on [value auto-infer](https://windicss.org/features/value-auto-infer.html#value-auto-infer), while keeping the code to a minimal size.
